src/backend/commands/modulecmds.c

```diff
@@ -304,6 +304,72 @@ AlterModuleAlterFunction(ParseState *pstate, AlterModuleAlterFuncStmt *stmt)
 	return address;
 }
 
+/*
+ * Rename module
+ */
+ObjectAddress
```

Comment:
I'm not certain, but it looks like this function never updates the system catalog.

Reply:
I'm calling CatalogTupleUpdate() to update the single tuple for this module in the pg_module catalog. Is there something else you were thinking we might need to do?